
Header header

uint16 sequence

Object[] objects

